docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,32 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
+
+Before reading the following, also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Does your content really need images?
+Generally, we would avoid adding new images to our content.
+
+In patterns and components, we should be using code examples when we want to convey what our pattern or component will look like. Avoid using screenshots of patterns or components wherever possible.
+
+## Can your image be an SVG?
+SVG's are usually better for web performance as they scale with different screen sizes and load faster than most image formats of the same size. If your image is a "graphic" (in other words it isn't a photo or a screenshot) it should be converted to an SVG and added to the page directly.
+
+This doesn't however mean that SVG's can't also have a negative impact on performance if their file size is unnecessarily large. Consider using a compression service such as [SVGOMG](https://jakearchibald.github.io/svgomg/) to ensure your SVG is as performant as possible.
+
+## Ensure your image is an appropriate file size
+If your image can't or shouldn't be an SVG is instead a "standard" file type eg: PNG, JPEG etc, avoid adding very large file size images to the website.
+
+The simplest way to avoid this is is to compress your images. There are numerous compression tools such as [ImageOptim](https://imageoptim.com/mac) which can handle this for you.
+
+Another good practice is to ensure that your image is the appropriate dimensions. For example, if your image is going to sit within a width container or column on a page and you know that the maximum size your image will ever be, accounting for different screen sizes, is 500 pixels, your image's intrinsic size (its "actual" size) should reflect this. A good rule of thumb is for your image's intrinsic width to be double that of it's maximum width on the page. In the previous example, we would want our image to have a width of 1000px. This is to account for HiDPI (High Dots Per Inch) displays such as retina display macs where lower-pixel images may look blurry and hard to interpret. There are numerous tools for resizing images, such as within macOS's own image viewer.
```

Comment:
My personal experience is that 2x resolution is usually not necessary, even on high dpi displays. I've found both of these scenarios to be basically the same and produce good results:

* 1.5x resolution and low compression
* 2x resolution but with agressive compression.

Reply:
It's absolutely the case that if a service has 'retina' graphics, then you can have much heavier compression than if you don't.

Reply:
Suppose that advice came as a compromise through lack of `srcset` or `<picture>` support?

E.g. Add extra width (1.5x, 2x) to your images to avoid blurriness on high-DPI devices

But ideally we'd like browsers pick the appropriate scaling size using `srcset`

```html
<img
  src="example.png"
  srcset="example.png, example@2x.png 2x, example@3x.png 3x"
  alt="A screenshot showing an example."
>
```

Or similarly the `<picture>` example above in https://github.com/alphagov/govuk-design-system/pull/2410#discussion_r1212004990

```html
<picture>
  <source type="image/webp" srcset="example.webp, example@2x.webp 2x, example@3x.webp 3x">
  <img src="example.jpg" alt="A photo better suited to WebP compression, but JPEG will do.">
</picture>
```

We had a chat through these options in https://github.com/alphagov/govuk-design-system/pull/2410#discussion_r1104403168 and linked to https://web.dev/high-dpi/

Using 1.5x is a quick way to support Windows "Scale" modes 125%, 150% without the file size of 2x though